### ISSUE TITLE
Alterar experiência de usuário para coletar email do participante da coleta

### ISSUE THREAD

**Issue by davidCarlos** [cidadedemocratica/ej-server#171]
O componente de coleta possui três mecanismos de autenticação. Um desses mecanismos é um formulário de cadastro, que aparece antes do usuário iniciar a participação na conversa.

Temos notado uma baixa taxa de conversão quando o componente é executado com o mecanismo de autenticação via formulário de cadastro, enquanto que via mecanismo de autenticação mautic, que o usuário não precisa informar nenhum campo pessoal, a taxa de conversão é bastante alta.

Essa issue consiste em alterar o momento em que o formulário de cadastro aparece. A ideia é que o usuário se autentique via o cookie do analytics ou do mautic (que não exige informações do usuário), e após um certo número de votos o componente de registro apareça. O usuário poderá não informar o campo de email, e continuar participando da coleta.

**Comment by davidCarlos** [comment]
@ricardopoppi é +- isso que tinhamos discutido né?

**Comment by ricardopoppi** [comment]
> @ricardopoppi é +- isso que tinhamos discutido né?

exatamente. eu só ajustei direto no corpo da issue o termo pra "aquisição" pq é como a gente tá tentando padronizar
